qradar/release_notes.md

```diff
@@ -1,4 +1,13 @@
-#### What's Fixed
-  
-- Updated jinja .timestamp to .int_timestamp in the data ingestion playbooks.
-- Added the "PATCH" option to the "Request Method" parameter of the "Invoke QRadar REST API" operation.
\ No newline at end of file
+#### What's new in 1.6.0
+
+- New actions:
+    - Get assets properties: Get a list of available asset property types that can be used with assets update operation
```

Comment:
just keep the Action name. Description is not needed.

Reply:
done